src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+.form-field {
+  margin-bottom: 20px;
+}
+
+.form-input-gap {
+  margin-bottom: 10px;
+}
```

Comment:
You may only apply classes only to your `filedset` and form field (`label` or `div`)
Example: 
```
.fieldset:not(:last-child) {
  margin-bottom: 20px
}

.field:not(:last-child) {
  margin-bottom: 10px
}
```